educe/learning/edu_input_format.py

```diff
@@ -28,13 +31,16 @@ def _dump_edu_input_file(docs, f):
             edu_txt = edu.text().replace('\n', ' ')
             # subgroup: sentence identifier, backoff on EDU id
             sent_idx = edu2sent[i]
-            subgroup = ('{}_sent{}'.format(grouping, sent_idx)
-                        if sent_idx is not None
-                        else edu_gid)
+            if sent_idx is None:
+                subgroup = edu_gid
+            elif isinstance(sent_idx, six.string_types):
```

Comment:
You are right in that `edu2sent` is quite low-level and should probably not be accessed here.
Maybe `DocumentPlus` objects should have a method to export the information necessary for a dump, or something in this vein.
I propose we think about this in the coming weeks and I accept the PR for now.
Are you good with this course of action?